index.html

```diff
@@ -300,6 +175,193 @@ <h3 class="about-myself-connect">LET'S CONNECT</h3>
+              <li><a href="#">javascript</a></li>
+            </ul>
+            <hr class="line" />
+          </div>
+
+          <div class="btn">
+            <button type="submit" class="btn-tonic-single">See live</button>
+            <button type="submit" class="btn-tonic-single">See Code</button>
+          </div>
+        </div>
+        <div class="end-bar">
+          <div class="line">
+            <img src="/line.png" alt="" />
+          </div>
+        </div>
+      </section>
+
+      <section class="second-card-section hidden" data-popup-id="myPopup-1">
+        <div class="all-components">
+          <div class="tonic-content">
+            <div class="title-tonic">
+              <h1>Tonic</h1>
+              <i class="fas fa-times close-popup" data-remove-popup="0"></i>
+            </div>
+            <div class="tonic-content-canopy">
+              <p>CANOPY</p>
+              <ul class="tonic-list">
+                <li>Back End Dev</li>
+                <li>2021</li>
+              </ul>
+            </div>
+            <div class="first-image">
+              <img src="./asset/Snapshoot(1).png" alt="" />
+            </div>
+            <p>
+              Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a galley of
+              type and scrambled it to make a type specimen book. It has
+              survived not only five centuries, but also the leap into
+              electronic typesetting, remaining essent
+            </p>
+            <div class="tonic-courses">
+              <ul>
+                <li><a href="#">html</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">javascript</a></li>
+              </ul>
+              <hr class="line" />
+            </div>
+
+            <div class="btn">
+              <button type="submit" class="btn-tonic">See live</button>
+              <button type="submit" class="btn-tonic">See Code</button>
+            </div>
+          </div>
+        </div>
+        <div class="end-bar">
+          <div class="line">
+            <img src="/line.png" alt="" />
+          </div>
+        </div>
+      </section>
+
+      <section class="second-card-section hidden" data-popup-id="myPopup-2">
+        <div class="all-components">
+          <div class="tonic-content">
+            <div class="title-tonic">
+              <h1>Tonic</h1>
+              <i class="fas fa-times close-popup" data-remove-popup="0"></i>
+            </div>
+            <div class="tonic-content-canopy">
+              <p>CANOPY</p>
+              <ul class="tonic-list">
+                <li>Back End Dev</li>
+                <li>2021</li>
+              </ul>
+            </div>
+            <div class="first-image">
+              <img src="./asset/Snapshoot(2).png" alt="" />
```

Comment:
- [ ] I suggest that you try to design the popup to follow the design for both desktop and mobile version from template in Figma. You can check out the comparison below and try to make changes accordingly

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/152639884-1626d71e-4e3e-4fd7-a60e-b1fe7b7e306b.png) | ![image](https://user-images.githubusercontent.com/4247740/152639959-9621b119-b95a-43df-85e2-272c89ed679c.png) |